3_database.py

```diff
@@ -0,0 +1,357 @@
+# In[5]:
+
+
+df_players = open_file("players")
+df_players.head()
+
+
+# ## Database creation
+
+# In[6]:
+
+
+def create_DB(nombre_bbdd):
+    cnx = mysql.connector.connect(user='root', password='AlumnaAdalab',
+                                host='127.0.0.1')
+
+    mycursor = cnx.cursor()
+
+    sql = (f"CREATE DATABASE {nombre_bbdd}")
+    try:
+        mycursor.execute(sql)
+        print(f"The database {nombre_bbdd} has been successfully created.")
+
+    except mysql.connector.Error as err:
+        print(err)
+        print("Error Code:", err.errno)
+        print("SQLSTATE", err.sqlstate)
+        print("Message", err.msg)
+    return mycursor.close() 
+    
+
+
+# In[7]:
+
+
+create_DB("db_ball_api")
+
+
+# ## Tables creation
+
+# In[7]:
+
+
+# This is a Python function that connects to a MySQL database using the `mysql.connector` module and executes a given SQL query. 
+
+def execute_query(query): # The function takes a single argument `query`, which is the SQL query to be executed.
+    
+    cnx = mysql.connector.connect(user='root', password='AlumnaAdalab',
+                                host='127.0.0.1', database= 'DB_Ball_API')
+
+    mycursor = cnx.cursor() # It creates a cursor object, executes the query using the cursor, and commits the changes to the database.
+    
+    try: 
+        mycursor.execute(query)
+        cnx.commit()
+
+    except mysql.connector.Error as err:
+        print(err)
+        print("Error Code:", err.errno)
+        print("SQLSTATE", err.sqlstate)
+        print("Message", err.msg)
+    
+    else:
+        mycursor.close()
+        cnx.close() # Finally, the function closes the cursor and the database connection.
+
+
+# `Teams`
+
+# In[13]:
+
+
+teams_table_query = """CREATE TABLE IF NOT EXISTS table_teams (
+id INT NOT NULL,
+abbreviation VARCHAR(255),
+city VARCHAR(255),
+conference VARCHAR(255),
+division VARCHAR(255),
+full_name VARCHAR(255),
+name VARCHAR(255),
+PRIMARY KEY (id));"""
+
+
+# In[14]:
+
+
+execute_query(teams_table_query)
+
+
+# `Games`
+
+# In[15]:
+
+
+games_table_query = """CREATE TABLE IF NOT EXISTS table_games (
+id INT NOT NULL ,
+`date` VARCHAR(255) ,
+home_team_score INT ,
+period INT ,
+postseason VARCHAR(255) ,
+season INT ,
+status VARCHAR(255) ,
+time VARCHAR(255) ,
+visitor_team_score INT ,
+`home_team.id` INT ,
+`home_team.abbreviation` VARCHAR(255) ,
+`home_team.city` VARCHAR(255) ,
+`home_team.conference` VARCHAR(255) ,
+`home_team.division` VARCHAR(255) ,
+`home_team.full_name` VARCHAR(255) ,
+`home_team.name` VARCHAR(255) ,
+`visitor_team.id` INT ,
+`visitor_team.abbreviation` VARCHAR(255) ,
+`visitor_team.city` VARCHAR(255) ,
+`visitor_team.conference` VARCHAR(255) ,
+`visitor_team.division` VARCHAR(255) ,
+`visitor_team.full_name` VARCHAR(255) ,
+`visitor_team.name` VARCHAR(255) ,
+PRIMARY KEY (id));"""
+
+
+# In[16]:
+
+
+execute_query(games_table_query)
+
+
+# `Players`
+
+# In[17]:
+
+
+players_table_query = """CREATE TABLE IF NOT EXISTS table_players (
```

Comment:
instead of hardcoding the definition in a string, could you reference the file in `sql/ddl/` and read it from there? We don´t want to replicate the same code twice

Reply:
for that, maybe you need to create one sql file for each table